README.md

```diff
@@ -1,5 +1,8 @@
-# Mattermost Heroku
+# Deploy Mattermost Team or Enterprise Edition to Heroku
+
+[![Deploy](https://www.herokucdn.com/deploy/button.svg)](https://heroku.com/deploy)
+
+Updated version of Mattermost to deploy in HEROKU. Updated by @2CodersTeam
```

Comment:
I think you dont need to set this `Updated by @2CodersTeam`

any comments @jwilander ?

Reply:
no problem... just removed